drf_extra_fields/fields.py

```diff
@@ -24,11 +24,14 @@
 from rest_framework.utils import html
 from drf_extra_fields import compat
 
-
 DEFAULT_CONTENT_TYPE = "application/octet-stream"
 
 
 class Base64FieldMixin(object):
+    trust_provided_content_type = False
```

Comment:
This should be provided via __init__ method. So that you don't need your own class to change this. 

Similar to `represent_in_base64` usage.